tests/unit/staff/views/test_users.py

```diff
@@ -560,6 +560,12 @@ def test_userlist_filter_by_any_roles_no_excludes_project_roles(rf, core_develop
     user_with_project_and_no_roles = UserFactory()
     ProjectMembershipFactory(user=user_with_project_and_no_roles)
 
+    user_with_mixture_of_project_roles = UserFactory()
```

Comment:
Can you add the similar for the `test_userlist_filter_by_any_roles_no_excludes_org_roles` and create a `user_with_mixture_of_org_roles`? This should check the change you've made to count the number of org roles.